__tests__/app.test.js

```diff
@@ -68,4 +68,42 @@ describe("GET /api", () => {
         });
     });
   });
+
+  describe("GET /api/articles/:article_id", () => {
+    test("200: Responds with an article object with correct properties", () => {
+      return request(app)
+        .get("/api/articles/1")
+        .expect(200)
+        .then(({ body: { article } }) => {
+          expect(article).toMatchObject({
+            author: expect.any(String),
+            title: expect.any(String),
+            article_id: expect.any(Number),
+            body: expect.any(String),
+            topic: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            article_img_url: expect.any(String),
+          });
```

Comment:
You are writing a test to see if article 1 comes back from the server. This test would pass if any article came back. You absolutely should be more explicit